[Snyk] Security upgrade alpine from 3.7 to 3.18.12

TheRedHatter opened the pull request:
![snyk-top-banner](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/pr-banner-default.svg)

### Snyk has created this PR to fix 1 vulnerabilities in the dockerfile dependencies of this project.

Keeping your Docker base image up-to-date means you’ll benefit from security fixes in the latest version of your chosen image.

#### Snyk changed the following file(s):

- `installer/resources/pacbot_app/files/ui_docker/dockerfile`

We recommend upgrading to `alpine:3.18.12`, as this image has only **0** known vulnerabilities. To do this, merge this pull request, then verify your application still works as expected.



#### Vulnerabilities that will be fixed with an upgrade:

|  | Issue | Score | 
:-------------------------:|:-------------------------|:-------------------------
![critical severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/c.png 'critical severity') | Out-of-bounds Write <br/>[SNYK-ALPINE37-MUSL-458286](https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286) | &nbsp;&nbsp;**714**&nbsp;&nbsp; 
![critical severity](https://res.cloudinary.com/snyk/image/upload/w_20,h_20/v1561977819/icon/c.png 'critical severity') | Out-of-bounds Write <br/>[SNYK-ALPINE37-MUSL-458286](https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286) | &nbsp;&nbsp;**714**&nbsp;&nbsp; 



---

> [!IMPORTANT]
>
> - Check the changes in this PR to ensure they won't cause issues with your project.
> - Max score is 1000. Note that the real score may have changed since the PR was raised.
> - This PR was automatically created by Snyk using the credentials of a real user.

---

**Note:** _You are seeing this because you or someone else with access to this repository has authorized Snyk to open fix PRs._

For more information: <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6ImhEUUhEb0w0UDlMQUI1NWlqVnpPc0dHYjM0NHhsTGc5IiwiYW5vbnltb3VzSWQiOiJiMWQyZGNjYi03Y2Y4LTQwYjItYTkzMC1hMzcxZjY4MThhMTQiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6ImIxZDJkY2NiLTdjZjgtNDBiMi1hOTMwLWEzNzFmNjgxOGExNCJ9fQ==" width="0" height="0"/>
🧐 [View latest project report](https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/project/ff3c83d5-04cc-45ee-83e9-368ad82712ef?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr)
📜 [Customise PR templates](https://docs.snyk.io/scan-using-snyk/pull-requests/snyk-fix-pull-or-merge-requests/customize-pr-templates?utm_source=github&utm_content=fix-pr-template)
🛠 [Adjust project settings](https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/project/ff3c83d5-04cc-45ee-83e9-368ad82712ef?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;fix-pr/settings)
📚 [Read about Snyk's upgrade logic](https://docs.snyk.io/scan-with-snyk/snyk-open-source/manage-vulnerabilities/upgrade-package-versions-to-fix-vulnerabilities?utm_source=github&utm_content=fix-pr-template)

---

**Learn how to fix vulnerabilities with free interactive lessons:**

🦉 [Learn about vulnerability in an interactive lesson of Snyk Learn.](https://learn.snyk.io/?loc&#x3D;fix-pr)

[//]: # 'snyk:metadata:{"breakingChangeRiskLevel":null,"FF_showPullRequestBreakingChanges":null,"FF_showPullRequestBreakingChangesWebSearch":null,"customTemplate":{"variablesUsed":[],"fieldsUsed":[]},"dependencies":[{"name":"alpine","from":"3.7","to":"3.18.12"}],"env":"prod","issuesToFix":["SNYK-ALPINE37-MUSL-458286","SNYK-ALPINE37-MUSL-458286"],"prId":"b1d2dccb-7cf8-40b2-a930-a371f6818a14","prPublicId":"b1d2dccb-7cf8-40b2-a930-a371f6818a14","packageManager":"dockerfile","priorityScoreList":[714],"projectPublicId":"ff3c83d5-04cc-45ee-83e9-368ad82712ef","projectUrl":"https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/project/ff3c83d5-04cc-45ee-83e9-368ad82712ef?utm_source=github&utm_medium=referral&page=fix-pr","prType":"fix","templateFieldSources":{"branchName":"default","commitMessage":"default","description":"default","title":"default"},"templateVariants":["updated-fix-title","priorityScore"],"type":"auto","upgrade":["SNYK-ALPINE37-MUSL-458286","SNYK-ALPINE37-MUSL-458286"],"vulns":["SNYK-ALPINE37-MUSL-458286"],"patch":[],"isBreakingChange":false,"remediationStrategy":"vuln"}'

snyk-bot committed:
fix: installer/resources/pacbot_app/files/ui_docker/dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286

AC-KunalParmar commented:
### :white_check_mark: **Snyk checks have passed. No issues have been found so far.**

| Status | Scan Engine | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-critical.svg) Critical | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg) High | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg) Medium | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg) Low | Total (0) |
|---|---|---|---|---|---|---|
| :white_check_mark: | Open Source Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/armorcode-partner/pr-checks/69645e80-31f7-438c-8050-a033944de6e8?refs=730486f4245244e48bacf48cca0b67f17eae5285&source=prChecksComment) |
| :white_check_mark: | Licenses| 0 | 0 | 0 | 0 | [ 0 issues](https://app.snyk.io/org/armorcode-partner/pr-checks/69645e80-31f7-438c-8050-a033944de6e8/license?refs=730486f4245244e48bacf48cca0b67f17eae5285&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

snyk-bot committed:
fix: installer/resources/pacbot_app/files/ui_docker/dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286

TheRedHatter commented:
### :white_check_mark: **Snyk checks have passed. No issues have been found so far.**

| Status | Scan Engine | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-critical.svg) Critical | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg) High | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-medium.svg) Medium | ![](https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-low.svg) Low | Total (0) |
|---|---|---|---|---|---|---|
| :white_check_mark: | Open Source Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/pr-checks/ee2f7d84-6ab0-4ec7-8016-9572ebb41f83?refs=730486f4245244e48bacf48cca0b67f17eae5285&source=prChecksComment) |
| :white_check_mark: | Licenses| 0 | 0 | 0 | 0 | [ 0 issues](https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/pr-checks/ee2f7d84-6ab0-4ec7-8016-9572ebb41f83/license?refs=730486f4245244e48bacf48cca0b67f17eae5285&source=prChecksComment) |
| :white_check_mark: | Code Security| 0 | 0 | 0 | 0 | [ 0 issues](https://app.us.snyk.io/org/armorcode-aws-partner-nfr-default/project/1722d236-fe18-4432-84b1-f77e5c0cbe1d/pr-check/bd8a1717-2892-4e0b-8611-41a66aa801b3?refs=730486f4245244e48bacf48cca0b67f17eae5285&source=prChecksComment) |

:computer: Catch issues earlier using the plugins for [VS Code](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner), [JetBrains IDEs](https://plugins.jetbrains.com/plugin/10972-snyk-security), [Visual Studio](https://marketplace.visualstudio.com/items?itemName=snyk-security.snyk-vulnerability-scanner-vs-2022), and [Eclipse](https://marketplace.eclipse.org/content/snyk-security#external-install-button).

snyk-bot committed:
fix: installer/resources/pacbot_app/files/ui_docker/dockerfile to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286
- https://snyk.io/vuln/SNYK-ALPINE37-MUSL-458286
